### DISCUSSION TITLE
205

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#191] [General]
# 潮流周刊第205期 - 听张信哲

封面图拍摄于上周末杭州大莲花听的张信哲的演唱会，整体体验挺好，都是朗朗上口的歌曲，不过长大了之后再听张信哲，突然感觉有些老实人的感觉，哈哈哈哈哈，不过人家马上 58 岁了，声音还是真好听的。

https://weekly.tw93.fun/posts/205-%E5%90%AC%E5%BC%A0%E4%BF%A1%E5%93%B2

**Comment by SunXin121** [comment]
PDFMathTranslate 的介绍写成 Awesome Cloudflare 的喽

**Reply by tw93** [reply]
谢谢反馈 已经修复

**Comment by XiaoLiz** [comment]
博主您好，您的电墨屏电子书阅读器是什么品牌

**Reply by Vainycos** [reply]
应该是kindle

**Comment by Vainycos** [comment]
同款kindle，猜的没错的话应该是kindle oais3？

**Reply by tw93** [reply]
掌阅抄的有点像了，看206期
